### DISCUSSION TITLE
Is AFL++ supported on Linux Redhat 8.7 ?

### DISCUSSION THREAD

**Discussion by Hongyuan-Li** [AFLplusplus/AFLplusplus#2080] [Q&A]
Is AFL++ supported on Linux Redhat 8.7 ? If it is true, how to build and install AFL ++ on Redhat 8.7 ? or do you have a Linux OS list for AFL++ supported now ? Thanks!

**Comment by vanhauser-thc** [comment]
Just follow docs/INSTALL.md

**Reply by Hongyuan-Li** [reply]
thanks, i follow the docs/INSTALL.md to build and install ATF++ but when get below error
[administrator@ECS-TEMPLATE ~]$ sudo yum install -y build-essential python3-dev automake cmake git flex bison libglib2.0-dev libpixman-1-dev python3-setuptools cargo libgtk-3-dev
[sudo] password for administrator:
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered with an entitlement server. You can use subscription-manager to register.

Last metadata expiration check: 1:00:20 ago on Fri 10 May 2024 05:39:10 PM PDT.
No match for argument: build-essential
No match for argument: python3-dev
Package flex-2.6.1-9.el8.x86_64 is already installed.
Package bison-3.0.4-10.el8.x86_64 is already installed.
No match for argument: libglib2.0-dev
No match for argument: libpixman-1-dev
Package python3-setuptools-39.2.0-7.el8.noarch is already installed.
No match for argument: libgtk-3-dev
Error: Unable to find a match: build-essential python3-dev libglib2.0-dev libpixman-1-dev libgtk-3-dev

**Reply by vanhauser-thc** [reply]
You can compile it on redhat but we don’t keep book what the package names are on each Linux type and version.
